.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit = **/migrations/*,**/settings/*,**/wsgi.py,**/tests/*,**/manage.py
+source = .
```

Comment:
Ah, that's worth knowing, thank you. Branch is a good idea, unfortunately it reduces coverage further to 22% :disappointed: